app/views/layouts/Layout.scala.html

```diff
@@ -32,14 +32,42 @@
 )
 @(pageTitle: String,
   scriptsBlock: Option[Html] = None,
-    isPageFullWidth: Boolean = true
+    isPageFullWidth: Boolean = true,
+    isUserAgent: Boolean = false,
+    newBreadCrumb: Option[String] = None
 )(contentBlock: Html)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @beforeContentBlock = {
     @phaseBanner("beta")
     @if(appConfig.welshLanguageSupportEnabled) {@languageSelect()}
 
-    @govukBackLink(BackLink(href = "#", content = Text(messages("app.common.back")), attributes = Map("id" -> "back-link")))
+    @if(isUserAgent) {
+        @govukBackLink(BackLink(href = "#", content = Text(messages("app.common.back")), attributes = Map("id" -> "back-link")))
+    } else {
+    <div class="govuk-breadcrumbs govuk-breadcrumbs--collapse-on-mobile">
+        <ol class="govuk-breadcrumbs__list">
+            <li class="govuk-breadcrumbs__list-item">
+                <a class="govuk-breadcrumbs__link" href="@{appConfig.vatOverviewUrl}">
+                    @messages("index.breadcrumb.vat-account")
```

Comment:
Shouldn't the root be BTA, then the parent is VATVC and then us?
![image](https://user-images.githubusercontent.com/53828896/152333311-0ba0ed61-a187-40ed-ba12-37a876b2ba21.png)

Reply:
The prototype hasn't been updated with this yet is as I believe there is some communication going on about the format of the breadcrumbs and making sure we align with VATVC
There is a part 2 to this ticket which we can cover adding BTA to the breadcrumbs if required https://jira.tools.tax.service.gov.uk/browse/PRM-994